### PR TITLE
Add change log for #27651.

### PR DESCRIPTION
**Pull request by chenrujun**
As title. Add change log for #27651

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @saragluna , @yiliuTo , please help to review this PR when you have time.

File: sdk/spring/CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ This section includes changes in `spring-cloud-azure-stream-binder-servicebus` m
 - Change property from `spring.cloud.stream.servicebus.bindings.<binding-name>.consumer.session-aware` to `spring.cloud.stream.servicebus.bindings.<binding-name>.consumer.session-enabled` [#27331](https://github.com/Azure/azure-sdk-for-java/pull/27331).
 - Unify the root package name of Spring libraries. [#27420](https://github.com/Azure/azure-sdk-for-java/pull/27420).
 - Remove message header of `AzureHeaders.RAW_ID`. Please use `ServiceBusMessageHeaders.MESSAGE_ID` instead [#27675](https://github.com/Azure/azure-sdk-for-java/pull/27675).
+- `KeyVaultPropertySource`'s configuration properties changed. [27651](https://github.com/Azure/azure-sdk-for-java/pull/27651)
```
Comment (saragluna) [inline sdk/spring/CHANGELOG.md:24]: we should add a section for `### Spring Cloud Azure AutoConfigure` and put these under the breaking changes of that section

Comment (chenrujun) [reply sdk/spring/CHANGELOG.md]: Updated.

File: sdk/spring/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,17 @@
 ### Dependency Updates
 Upgrade dependency according to spring-boot-dependencies:2.6.3 and spring-cloud-dependencies:2021.0.0.
 
+### Spring CLoud Azure AutoConfigure
```
Comment (saragluna) [inline sdk/spring/CHANGELOG.md:10]: Autoconfigure

**Review by saragluna — CHANGES_REQUESTED** [review]

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**